examples/02-plot/strips-example.py

```diff
@@ -0,0 +1,25 @@
+"""
+Build a figure from strips
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Keep this so we can use the sphinx `ref` directive. Also, check title length.

```suggestion
.. _strips_example:

Build a figure from strips
~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
This suggestion `.._strips_example` seems to cause a failure while building the documentation. I'm just following the documentation from other examples, so I'm not sure why this is failing.

Reply:
You are missing the space between `..` and `_strips_example`. For future reference, the way @akaszynski posted the suggestion you can click the "commit suggestion" button here to apply the change directly if you want to (then you don't have to copy the change manually).